parsec/api/protocol/organization.py

```diff
@@ -96,16 +97,41 @@ class APIV1_OrganizationStatusReqSchema(BaseReqSchema):
 class APIV1_OrganizationStatusRepSchema(BaseRepSchema):
     is_bootstrapped = fields.Boolean(required=True)
     expiration_date = fields.DateTime(allow_none=True, required=False)
+    outsider_enabled = fields.Boolean(required=True)
```

Comment:
@vxgmichel 
![image](https://user-images.githubusercontent.com/3187637/120362370-48e5ac00-c30b-11eb-8c55-bde9517c46b2.png)

Reply:
`user_profile_outsider_allowed` sounds ok, maybe `is_outsider_profile_allowed` or `allow_outsider_profile` for consistency.

Reply:
@vxgmichel @touilleMan what is your final answer ?

Reply:
```
$ rg Boolean parsec/api/
parsec/api/protocol\realm.py
57:    in_maintenance = fields.Boolean(required=True)

parsec/api/protocol\events.py
65:    wait = fields.Boolean(missing=True)

parsec/api/protocol\organization.py
97:    is_bootstrapped = fields.Boolean(required=True)

parsec/api/protocol\invite.py
42:    send_email = fields.Boolean(required=True)
47:    send_email = fields.Boolean(required=True)

parsec/api/data\certif.py
56:        is_admin = fields.Boolean(required=True)

parsec/api/protocol\user.py
52:    omit_revoked = fields.Boolean(missing=False)
260:    omit_revoked = fields.Boolean(missing=False)
261:    omit_non_human = fields.Boolean(missing=False)
269:    revoked = fields.Boolean(required=True)
```

There is no strong consistency between the `is_spammed`, `is_spam` and `spammed` formats...

`allow_outsider_profile` brings yet another format so I'd say to avoid it

In the end I would stick with `user_profile_outsider_allowed` given it's the most explicit (in the future we may have very different options such as `workspace_stats_allowed_roles = fields.List(choices=RealmRole, allow_none=True)` so better be explicit ^^)